src/components/Comment/Comment.tsx

```diff
@@ -123,31 +103,31 @@ const flexRowStyles = css`
   flex-direction: row;
 `;
 
-const removePaddingLeft = css`
-  padding-left: 0px;
+const flexColStyles = css`
+  display: flex;
+  flex-direction: column;
 `;
 
-const Column = ({ children }: { children: JSX.Element | JSX.Element[] }) => (
-  <div
-    className={css`
-      display: flex;
-      flex-direction: column;
-    `}
-  >
-    {children}
-  </div>
-);
+const alignItemsCenter = css`
+  align-items: center;
+`
 
-const Row = ({ children }: { children: JSX.Element | JSX.Element[] }) => (
-  <div
-    className={css`
-      display: flex;
-      flex-direction: row;
-    `}
-  >
-    {children}
-  </div>
-);
+const fullWidthStyles = css`
+  width: 100%;
+`
+
+const spaceBetweenStyles = css`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const headerStyles = css`
```

Comment:
Is this redundant if we already have `spaceBetweenStyles`?

Reply:
just missed this, thanks for pointing out